distribute-with-merkl/create-a-campaign.md

```diff
@@ -100,3 +100,58 @@ Once you've executed your transaction, your campaign may take up to **one hour**
 {% hint style="note" %}
 It's expected for your campaign's **TVL and APR to show 0** at launch, and for the **leaderboard to appear empty**. The Merkl Engine needs a few hours to perform [the necessary computations](../merkl-mechanisms/technical-overview.md#reward-computation) before this data becomes available.
 {% endhint %}
+
+## Creating campaigns on behalf of a creator
```

Comment:
can you put this in the before you start section: as helpful as well in the context of the multi campaign section